FAILED test/unit/deadline_adaptor_for_maya/MayaClient/test_client.py::TestMayaClient::test_mayaclient - AttributeError: <module 'deadline.maya_adaptor.MayaClient.maya_client' from '/Users/runner/work/deadline-cloud-for-maya/deadline-cloud-for-maya/src/deadline/maya_adaptor/MayaClient/maya_client.py'> does not have the attribute 'HTTPClientInterface'
FAILED test/unit/deadline_adaptor_for_maya/MayaClient/test_client.py::TestMayaClient::test_main - AttributeError: <module 'deadline.maya_adaptor.MayaClient.maya_client' from '/Users/runner/work/deadline-cloud-for-maya/deadline-cloud-for-maya/src/deadline/maya_adaptor/MayaClient/maya_client.py'> does not have the attribute 'HTTPClientInterface'

--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -308,18 +308,47 @@ or modifying functionality, then you will want to be writing one or more integ t
 logic behaves as expected and that future changes do not accidentally break your change.
 
 To run the integ tests, you need to:
-1. Add `mayapy` executable to the PATH:
+1. Install Maya 2025
+2. Install Redshift 2025
+3. Add `mayapy` executable to the PATH:
    
    For example, default path of mayapy on OSX is 
 
    ```
    /Applications/Autodesk/maya<version>/Maya.app/Contents/MacOS
    ```
-2. Use hatch to run all adaptor tests:
+4. Navigate to your local checkout of this repository:
+   ```
+   cd </path/to/deadline-cloud-for-maya>
+   ```
+5. Install your local `deadline-cloud-for-maya` repository into Maya's Python environment:
+   ```
+   mayapy -m pip install -e . --force-reinstall
+   ```
+6. Install the testing dependencies into Maya's Python environment:
+   ```
+   mayapy -m pip install -r requirements-testing.txt
+   ```
+7. Install the integration testing dependencies into Maya's Python environment:
+   ```
+   mayapy -m pip install -r requirements-integ-testing.txt
+   ```
+8. Add Maya's Python's `bin/` folder to your `PATH`:
+
+   For example, the default path on OSX is
+   ```
+   /Applications/Autodesk/maya2025/Maya.app/Contents/Frameworks/Python.framework/Versions/Current/bin/
+   ```
+
+   So you can run:
+   ```
+   export PATH="/Applications/Autodesk/maya2025/Maya.app/Contents/Frameworks/Python.framework/Versions/Current/bin/:$PATH"
+   ```
+9. Use hatch to run all adaptor tests:
    ```bash
    hatch run integ:test_adaptors_all
    ```
-3. Use hatch to run adaptor tests for a specific maya renderer plugin: 
+10. Use hatch to run adaptor tests for a specific maya renderer plugin:
    ```bash
    hatch run integ:test_adaptors_maya
    ```
@@ -328,7 +357,7 @@ To run the integ tests, you need to:
    hatch run integ:test_adaptors_redshift
    ```
 
-4. (Optional) Use hatch to run all integ tests:
+11. (Optional) Use hatch to run all integ tests:
    ```bash
    hatch run integ:test
    ```
@@ -341,7 +370,7 @@ To run the integ tests, you need to:
 #### Build the package
 
 ```bash
-hatch run build
+hatch build
 ```
 
 #### Build the installer

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,7 +31,7 @@ classifiers = [
 
 dependencies = [
     "deadline == 0.51.*",
-    "openjd-adaptor-runtime == 0.8.*",
+    "openjd-adaptor-runtime == 0.9.*",
 ]
 
 [project.urls]

--- a/src/deadline/maya_adaptor/MayaClient/maya_client.py
+++ b/src/deadline/maya_adaptor/MayaClient/maya_client.py
@@ -9,14 +9,9 @@ from typing import Optional
 # The Maya Adaptor adds the `openjd` namespace directory to PYTHONPATH,
 # so that importing just the adaptor_runtime_client should work.
 try:
-    from adaptor_runtime_client import HTTPClientInterface  # type: ignore[import]
+    from adaptor_runtime_client import ClientInterface  # type: ignore[import]
 except (ImportError, ModuleNotFoundError):
-    try:
-        from openjd.adaptor_runtime_client import HTTPClientInterface  # type: ignore[import]
-    except (ImportError, ModuleNotFoundError):
-        # TODO: Remove this try/except once we bump to openjd.adaptor_runtime_client 0.9+
-        # On Windows, HTTPClientInterface is not available, only ClientInterface
-        from openjd.adaptor_runtime_client import ClientInterface as HTTPClientInterface  # type: ignore[import]
+    from openjd.adaptor_runtime_client import ClientInterface  # type: ignore[import]
 
 
 try:
@@ -29,7 +24,7 @@ except (ImportError, ModuleNotFoundError):
     )
 
 
-class MayaClient(HTTPClientInterface):
+class MayaClient(ClientInterface):
     def __init__(self, server_path: str) -> None:
         super().__init__(server_path=server_path)
         self.actions.update(

--- a/test/unit/deadline_adaptor_for_maya/MayaClient/test_client.py
+++ b/test/unit/deadline_adaptor_for_maya/MayaClient/test_client.py
@@ -9,7 +9,7 @@ from deadline.maya_adaptor.MayaClient.maya_client import MayaClient, main
 
 
 class TestMayaClient:
-    @patch("deadline.maya_adaptor.MayaClient.maya_client.HTTPClientInterface")
+    @patch("deadline.maya_adaptor.MayaClient.maya_client.ClientInterface")
     def test_mayaclient(self, mock_httpclient: Mock) -> None:
         """Tests that the maya client can initialize, set a renderer and close"""
         client = MayaClient(server_path=str(9999))
@@ -19,7 +19,7 @@ class TestMayaClient:
     @patch("deadline.maya_adaptor.MayaClient.maya_client.os.path.exists")
     @patch.dict(os.environ, {"MAYA_ADAPTOR_SERVER_PATH": "server_path"})
     @patch("deadline.maya_adaptor.MayaClient.MayaClient.poll")
-    @patch("deadline.maya_adaptor.MayaClient.maya_client.HTTPClientInterface")
+    @patch("deadline.maya_adaptor.MayaClient.maya_client.ClientInterface")
     def test_main(self, mock_httpclient: Mock, mock_poll: Mock, mock_exists: Mock) -> None:
         """Tests that the main method starts the maya client polling method"""
         # GIVEN

